ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.view.js

```diff
@@ -491,6 +494,14 @@ module.exports = Marionette.LayoutView.extend({
     this.mapModel.addDistanceInfo(distanceInfoView)
     this.distanceInfo.show(distanceInfoView)
   },
+  setupPopupPreview() {
+    this.popupPreview.show(
+      new PopupPreviewView({
+        map: this.map,
+        mapModel: this.mapModel,
```

Comment:
Pass in the selectionInterface here, otherwise this won't work for routes beyond the workspace route.